migrations/20170628180932-add-demo-config-to-site.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function up(db, callback) {
+  db.addColumn('site', 'demoConfig', 'text', (addColErr) => {
+    if (addColErr) {
+      callback(addColErr);
+    } else {
+      db.runSql('UPDATE site SET "demoConfig" = config', (runErr) => {
```

Comment:
👍  Confirming that the demo branch currently uses the site config so this should be the column we want